src/core/profiles/dto/add-language.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsInt, IsString } from 'class-validator';
+
+export class AddLanguageDto {
+  @ApiProperty()
+  @IsInt()
+  languageId: number;
+
+  @ApiProperty()
+  @IsString()
+  level: string;
```

Comment:
Deberia ser un ENUM y limitar el nivel a las opciones que le sistema permita, evaluar con @Nannajpg si usaremos A1, A2, B1, B2, etc... o mas bien "Basico" "Intermedio" "Avanzado" "Nativo" ,etc...
```suggestion
  level: string;
```

Reply:
PD: seria bueno hacer que sea nulo.

Reply:
Bueno esto lo tenemos que preguntar si quieres hacerlo un ENUM pq yo no se si los niveles A1, B1 se usan en todos los idiomas o solo en ingles